Mage.Tests/src/test/java/org/mage/test/cards/single/clb/AbolethSpawnTest.java

```diff
@@ -0,0 +1,53 @@
+package org.mage.test.cards.single.clb;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ * @author Rowan-Gudmundsson
+ */
+public class AbolethSpawnTest extends CardTestPlayerBase {
```

Comment:
(Optional) Consider adding Soul Warden to Player B in this test, and confirming that Aboleth Spawn doesn't offer to copy Soul Warden's trigger.